xml/quick_configuration.xml

```diff
@@ -156,22 +165,38 @@
         <term>Certificate</term>
          <listitem>
           <para>
+           The <guimenu>Certificate</guimenu> field must be a 
+           Base64-encoded PEM key.
+          </para>
+          <note>
+           <title>Remove the enclosing lines</title>
+           <para>
+            You must enter the certificate body only. This means
+            only the content between the <literal>-----BEGIN CERTIFICATE-----</literal> 
+            and <literal>-----END CERTIFICATE-----</literal> lines
+            but not those lines themselves.
           </para>
+         </note>
          </listitem>
         </varlistentry>
         
         <varlistentry>
          <term>Anonymous</term>
           <listitem>
            <para>
+            Selects if you wish to perform an anonymous bind with
+            the LDAP server. If Set to <literal>False</literal> you
+            must provide a <guimenu>DN</guimenu>.
            </para>
           </listitem>
          </varlistentry>
          
          <varlistentry>
+          <!-- Is this field shown when Anonymous is set to True? -->
           <term>DN</term>
            <listitem>
             <para>
+             Distinguished Name used for the authentication.
```

Comment:
Can you give me a short description what the required DN fields will look like?
Which ones are required?

Reply:
Syntax for a DN (distinguished name) is very flexible.  [Section 4](https://tools.ietf.org/html/rfc4514.html) of this RFC gives some examples.  A common thread among the examples is ",dc=domain,dc=tld" but this can be preceded by many combinations of "cn", "uid" and "ou" values.  It's all up to how the LDAP server is structured and configured by the end user.  Since these LDAP servers will often predate a CaaSP cluster, we must be able to accept the full range allowed by the syntax.